maxim/apis/maxim_apis.py

```
@@ -109,47 +108,194 @@ def __init__(self, base_url: str, api_key: str):
+                            != current_version
+                        ):
+                            latest_version = response.headers["x-lt-maxim-sdk-version"]
+                            latest_version_parts = list(
+                                map(int, latest_version.split("."))
+                            )
+                            current_version_parts = list(
+                                map(int, current_version.split("."))
+                            )
+                            if latest_version_parts > current_version_parts:
+                                scribe().warning(
+                                    f"\033[33m[MaximSDK] SDK version is out of date. Please update to the latest version. Current version: {current_version}, Latest version: {latest_version}\033[0m",
```

Pylint (3.3.7)

[convention] Line too long (199/100)

(C0301)